### DISCUSSION TITLE
Marketing prompts

### DISCUSSION THREAD

**Discussion by lDrakohh** [ispp-2324-ocial/KB#5] [Prompts IAS]
Insert here prompts use to generate marketing content

**Comment by adrrf** [comment]
Se le ha pedido a ChatGPT el contenido para un apartado de la landing page, más especificamente sobre que poner en el apartado para empresas. enlace: https://chat.openai.com/share/47fce039-c093-48c2-8876-a97be73e93de

**Comment by paupenfer1** [comment]
Ejemplos de encuentras para medir rendimiento y satisfacción del equipo de trabajo https://chat.openai.com/share/2395920d-6d2a-4cb5-859e-12c35e043e95

**Comment by lDrakohh** [comment]
Se ha usado la IA fakeyou.com para generar una imagen en movimiento y la voz de Antonio Lobato.

**Comment by lDrakohh** [comment]
Se ha usado la ia SUNO.AI para generar una cancion para la app

https://app.suno.ai/song/1190787e-2fd7-4138-903a-0e6b37cf876e

**Comment by lDrakohh** [comment]
Se ha usado vidnoz AI para pasar texto a voz
https://es.vidnoz.com/texto-a-voz.html
